roles/fio_distributed/tasks/main.yml

```diff
@@ -2,8 +2,8 @@
 - block:
   - name: Provide path for fio - PV
     set_fact:
-      fio_path: /mnt/pvc
-    when: workload_args.storageclass is defined
+      fio_path: /dev/xvda
```

Comment:
So, if we don't use block is this still correct?

Reply:
meaning should we look at the type of storageclass and if it isn't block we keep the default to `/mnt/pvc` ?

Reply:
It worked for all 3 cases, pvcvolumemode is undefined, Block or Filesystem. When It's Filesystem /dev/xvda is considered as directory by FIO and for block /dev/xvda is considered as location of Block device.